dashboard/test/models/pd/workshop_material_order_test.rb

```diff
@@ -301,4 +301,18 @@ class Pd::WorkshopMaterialOrderTest < ActiveSupport::TestCase
     order.refresh
     assert_nil order.tracking_id
   end
+
+  test 'orders are still valid if the user is deleted' do
```

Comment:
As I said, no changes expected. But tests should test one and only one thing, IMO.